spec.bs

```diff
@@ -422,6 +422,30 @@ action of the user), then return.":
         {{DOMException}}.
     2. <a>Consume user activation</a> given <a>this</a>'s <a>relevant global object</a>.
 
+## Focusing the opener window ## {#focusing-the-opener-window}
+
+<p>
+It can often be useful for the picture-in-picture window to be able to re-focus
+its opener tab, e.g. when the smaller form-factor of the window doesn't fit the
+experience the user needs. We modify the {{Window/focus()}} API to allow it to
+take system-level focus when a picture-in-picture window is focusing its
+opener.
+</p>
+
+<p class="issue">
+Merge this into {{Window/focus()}} once it has enough consensus.
+</p>
+
+Add a new step to {{Window/focus()}} after step 3, "Run the <a>focusing steps</a> with |current|.":
+
+4. If |current| is a <a>top-level traversable</a>, then:
+    1. Let |pipWindow| be |current|'s <a>active window</a>'s
+        <a>documentPictureInPicture API</a>'s <a>last-opened window</a>.
+    2. If |pipWindow| is not <code>null</code> and |pipWindow|'s <a>relevant global object</a>
+        has <a>transient activation</a>, then:
```

Comment:
The problem is that focus() is still technically usable regardless, you just wouldn't get system-level focus in this case. It would essentially mean any time a window that owns a pip window calls window.focus() (or another window calls focus() on it), they'd get a new error despite otherwise having the same functionality as before